### PR TITLE
Migration cleanup for 4.2

### PR DESCRIPTION
**Pull request by apinnick**

### CODE REVIEW

**Comment by apinnick** [comment]
@bergerhoffer Ready for merge! Thanks

File: modules/migration-installing-cam-operator-ocp-3.adoc [hunk 1]
```diff
@@ -1,35 +1,33 @@
 // Module included in the following assemblies:
 //
 // migration/migrating_3_4/deploying_cam.adoc
+// Module included in the following assemblies:
```
Comment (bergerhoffer) [inline modules/migration-installing-cam-operator-ocp-3.adoc]: Duplicate metadata added

**Review by bergerhoffer — COMMENTED** [review]
Just one accidental duplication that needs to be removed.

File: modules/migration-installing-cam-operator-ocp-3.adoc (hunk 1 above)
Comment (apinnick) [reply modules/migration-installing-cam-operator-ocp-3.adoc]: ARGH... It's been a long day. Fixed.

**Comment by bergerhoffer** [comment]
Thanks, merging!

**Pull request merged**